packages/web/src/scss/components/Toast/_Toast.scss

```diff
@@ -134,27 +134,27 @@ $_queue-collapsed-visible-limit: 2; // 10.
 }
```

Comment:
Optional: replace `> *` with `> .ToastBar`.

Similarly, this could be possible:

```diff
-.Toast--collapsible > .Toast__queue > :where(:nth-last-child(-n + #{theme.$queue-collapse-after}))
+.Toast--collapsible > .Toast__queue > :where(.ToastBar:nth-last-child(-n + #{theme.$queue-collapse-after}))
```

etc.